neo/core/event.py

```diff
@@ -256,3 +258,48 @@ def as_quantity(self):
         Return the event times as a quantities array.
         """
         return self.view(pq.Quantity)
+
+    def to_epoch(self, pairwise=False, durations=None):
+        """
+        Transform Event to Epoch.
+
+        This method has three modes of action.
+
+        1. By default, an array of `n` event times will be transformed into
+           an array of `n-1` epochs, where the end of one epoch is the
+           beginning of the next.
+        2. If `pairwise` is True, then the event times will be taken as pairs
+           representing the start and end time of an epoch. The number of
+           events must be even, otherwise a ValueError is raised.
+        3. If `durations` is given, it should be a scalar Quantity or a
+           Quantity array of the same size as the Event.
+           Each event time is then taken as the start of an epoch of duration
+           given by `durations`.
+
+        `pairwise=True` and `durations` are mutually exclusive. A ValueError
+        will be raised if both are given.
+        """
```

Comment:
Here a comment should be made regarding the labels. Maybe:
If `durations` is specified, epoch labels are set to the corresponding labels of the events that indicate the epoch start. If `durations` is not specified, then the event labels A and B bounding the epoch are used to set the labels of the epochs in the form A-B.

Reply:
Done